base/applications/mspaint/lang/en-US.rc

```diff
@@ -262,4 +262,5 @@ BEGIN
     IDS_VERTICAL "Vertical"
     IDS_PRINTRES "%d x %d dots per inch"
     IDS_CANTPASTE "Failed to paste from the clipboard. The data format is either incorrect or not supported."
+    IDS_SAVEERROR "Failed to save an image as the following file:\n\n%s"
```

Comment:
"as the following file" ---> "in the following file" or "in the file"

Reply:
Fixed by @binarymaster at: https://github.com/reactos/reactos/commit/fa4c72b79e5592371cf8bdd218da3a8e03c0beff